custom/icds_reports/reports/incentive.py

```diff
@@ -91,13 +91,14 @@ def _format_infrastructure_data(data):
                     awh_eligible_for_incentive = 'No'
                 else:
                     num_open = row['awc_num_open']
+                    awh_eligible_for_incentive = 'Yes' if int(num_open) >= 21 else 'No'
```

Comment:
Does this change mean that when the feature flag is set, awh_eligible_for_incentive should be set strictly based on `awh_eligible` ? When the feature flag is released, will these two assignments be removed?

I think only setting awh_eligible_for_incentive in one place will be clearer

Reply:
Yeah, it goes from being set by a calculation applied to one column (checking size of num_open) to directly by a boolean column in the db (that was added in the last pr)

Reply:
So once this is released, the whole thing will be cleaned up to only being set by that column and we wont need to do any of the weird type checking to handle null values or casting ints